app/models/artist.rb

```diff
@@ -1,4 +1,6 @@
 class Artist < ActiveRecord::Base
+  has_many :songs
```

Comment:
One thing you might want to think about here is a destroy dependency.  If we delete an artist, do we want their songs to stick around, or do we want to delete them too?  (could go either way in this case)